code/src/Components/Reducers/todoitem.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const todoitem = createSlice({
```

Comment:
This is up to you, but since you have a component called ToDoItem, maybe the reducer can be called something else, like todos (since that's what we are storing) - to avoid mix ups.